src/commands/index.js

```diff
@@ -25,5 +26,6 @@ module.exports = {
   githubCommand,
   twitterCommand,
   reportCommand,
-  kontrolCommand
+  kontrolCommand,
+  hastebinizeCommand

```

Comment:
Trailing comma

Reply:
We're using standard.js in this project. Trailing comma is not allowed.

Reply:
I'll be damned, whatever floats your linter, then.